nva-datamodel-testutils/src/main/java/no/unit/nva/model/testing/PublicationGenerator.java

```diff
@@ -71,6 +77,18 @@ public static Publication randomPublication(Class<?> publicationInstanceClass) {
 
         return buildRandomPublicationFromInstance(publicationInstanceClass);
     }
+    public static Publication randomPublicationNonDegree() {
+        var degrees = Set.of(DegreeMaster.class,
+                             DegreeBachelor.class,
+                             DegreePhd.class,
+                             DegreeLicentiate.class,
+                             OtherStudentWork.class);
+        var nonDegrees =
+            PublicationInstanceBuilder.listPublicationInstanceTypes()
+                .stream().filter(i -> !degrees.contains(i)
+            ).collect(Collectors.toList());
```

Comment:
Would love to use a cleaner approch, but struggled with generics because listPublicationInstanceTypes() returns Class<?>.

Reply:
```
    public static Publication randomPublicationNonDegree() {
        var nonDegrees = PublicationInstanceBuilder.listPublicationInstanceTypes()
                .stream()
                .filter(not(PublicationGenerator::isDegree))
                .toList();
        return randomPublication(randomElement(nonDegrees));
    }

    private static boolean isDegree(Class<?> subClass) {
        return subClass.isAssignableFrom(DegreeBase.class);
    }
```

Reply:
Because we want the complement of the set of Degrees, we would have to do some jiggery pokery to get all classpath classes, so this is likely the best approach